statsmodels/genmod/tests/test_glm.py

```diff
@@ -80,6 +80,8 @@ def test_residuals(self):
                 resid_a, self.res1.resid_response))
         assert_allclose(resids, resid2, rtol=1e-6, atol=atol)
 
+        assert_allclose(self.res1.resid, self.res1.resid_response, rtol=1e-13)
```

Comment:
In principal this should be a separate method (I'll update the PR^2 to do this).  Besides which, the top of this test function has some comments suggesting something somewhere needs fixing; keeping narrow the code that comment refers to will make sorting that out easier.